components/fxa-client/src/internal/commands/close_remote_tabs.rs

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use base64::{engine::general_purpose::URL_SAFE_NO_PAD, Engine};
+use rc_crypto::ece;
+use serde_derive::*;
+
+use crate::{Error, Result, ScopedKey};
+
+use super::{
+    super::device::Device,
+    send_tab::{PrivateSendTabKeysV1, PublicSendTabKeys, SendTabKeysPayload},
+};
+
+pub const COMMAND_NAME: &str = "https://identity.mozilla.com/cmd/close-uris";
+pub const COMMAND_TTL: u64 = 30 * 24 * 3600;
```

Comment:
FxA has a "default TTL" for [well-known commands](https://github.com/mozilla/fxa/blob/c89a3b0be02eb36f8db7661595845de31dc62983/packages/fxa-auth-server/lib/routes/devices-and-sessions.js#L27-L30), but I captured it here explicitly because:

1. `close-uri` isn't a well-known command yet.
2. We might want to tweak the value. Does 30 days make sense?
3. We'll want this value to match whatever TTL we use for our "pending closed tabs" entry in the tabs engine. If the TTL passes and the receiving device still hasn't synced its tabs, we'll want to remove the closed tab from the tabs engine's "pending closed tabs" table.

Reply:
The TTL issue is interesting, but 30 days does seem excessive purely from the user's POV - closing a tab they requested be closed 30 days ago seems very likely to surpise the user? But no less surprising than a tab coming in, so I think this is probably fine in practice.